doc/website/sidebars.js

```diff
@@ -135,6 +135,18 @@ module.exports = {
                 },
             ],
         },
+        {
+            type: 'category',
+            label: 'SQLite',
+            collapsed: false,
+            items: [
+                {
+                    type: 'doc',
+                    id: 'guides/SQLite/partial-indexes-sqlite',
```

Comment:
keep urls and ids lowercased. 
```suggestion
                    id: 'guides/sqlite/partial-indexes-sqlite',
```